examples/step-91/doc/intro.dox

```diff
@@ -1267,11 +1268,53 @@ operator
 where $\alpha=K_d, \beta = k \left(\frac{p}{2}\right)^m$. If it weren't for the fact
 that the second term contains a derivative of $b(r)$, this would be what is known
 as a [Sturm-Liouville problem](https://en.wikipedia.org/wiki/Sturm%E2%80%93Liouville_theory).
-With the derivative, it doesn't fall into the same framework, but instead satisfies
-a variation of
-[Kummer's equation](https://en.wikipedia.org/wiki/Confluent_hypergeometric_function#Kummer's_equation)
+With the derivative, however, it doesn't fall into the same framework. Instead,
+a better way to see this equation is that it satisfies
+a variation of the
+[Kummer equation](https://en.wikipedia.org/wiki/Confluent_hypergeometric_function#Kummer's_equation),
 with solutions that were pointed out to us by Robert Israel
-[on this StackExchange post](https://math.stackexchange.com/questions/5025287/solution-of-a-nonsymmetric-eigenfunction-equation)
+[on this StackExchange post](https://math.stackexchange.com/questions/5025287/solution-of-a-nonsymmetric-eigenfunction-equation).
+More specifically, we can rearrange the equation for $b(r)$, multiply it by $r$, and divide by $K_d$ to read as
+@f[
+  r b''(r)
+  + \left(k \frac{1}{K_d} \left(\frac{p}{2}\right)^m r^{m+1} + 1 \right) b'(r)
+  + \frac{\lambda}{K_d} r b(r) = 0.
+@f]
+If we consider the case $m=0$ (i.e., erosion is independent of the
+amount of water available), then this equation *almost* (but not quite)
+has the same form as Kummer's equation
+mentioned on the Wikipedia page above, namely
+@f[
+  z\frac{d^2w}{dz^2} + (b-z)\frac{dw}{dz} - aw = 0,
+@f]
+if we replace $z\to r, w\to b, b\to 1, a\to \frac{\lambda}{K_d}$. But it isn't *exactly*
+the same because our equation still has the factor $r$ in front of $b(r)$, its
+sign is wrong, and the sign of the term $k \frac{1}{K_d} \left(\frac{p}{2}\right)^m r^{m+1} b'(r)$
+is also wrong. Yet, the equations are similarly enough that it is, then, perhaps not
+surprising that one can find some kind of transform than converts the two equations
```

Comment:
```suggestion
surprising that one can find some kind of transform that converts the two equations
```

Reply:
Thanks -- did you actually read through the whole document? ;-)

Reply:
Just through the last commit talking about Kummer functions.